### PR TITLE
[webui] Fix user show

### PR DESCRIPTION
**Pull request by ChrisBr**
The ordering in the array was wrong. First package, then project.

### CODE REVIEW

**Comment by ChrisBr** [comment]
@hennevogel please review!

**Comment by bgeuken** [comment]
Mind to write a test for this?;-)

**Comment by ChrisBr** [comment]
jop, will do that!

**Comment by ChrisBr** [comment]
@bgeuken added a test for it! Please review again

**Comment by bgeuken** [comment]
There seems to be a test failure right now.

The test looks good to me. To be honest I was having a simple unit test in mind, but this one looks great too:)

File: src/api/test/functional/search_controller_test.rb [hunk 1]
```diff
@@ -411,7 +411,7 @@ def test_find_owner
 
     get "/search/owner?binary='package'"
     assert_response 400
-    assert_xml_tag tag: 'status', :attributes => { :code => "attribute_not_set" }
+    assert_xml_tag tag: 'status', :attributes => { :code => "400", origin: "backend" }
```
Comment (ChrisBr) [inline src/api/test/functional/search_controller_test.rb]: We now have the OBS_OwnerRootProject attribute set on the Apache project!

**Comment by ChrisBr** [comment]
@bgeuken fixed the failing test!

**Comment by bgeuken** [comment]
Awesome!

LGTM

**Pull request merged**